torch/_inductor/fx_passes/mkldnn_fusion.py

```diff
@@ -1202,3 +1203,4 @@ def _mkldnn_weight_pack_init():
             _register_weight_pack_pass()
             _recover_linear()
             _register_quantization_weight_pack_pass()
+            _register_woq_lowerings()
```

Comment:
I feel a bit confused here. You are putting the woq passes here under `_mkldnn_weight_pack_init` but the implementation is actually **not** about weight packing but map the woq pattern to the respective kernel? Should this be moved to `_mkldnn_fusion_init`?

Reply:
Synced offline with Jiong. It is better to move `_register_woq_lowerings` to post grad, and in the future we could add the corresponding weight prepack in freezing passe, just like other OneDNN patterns.